packages/build-utils/src/types.ts

```diff
@@ -352,11 +352,41 @@ export interface BuilderV3 {
 
 type ImageFormat = 'image/avif' | 'image/webp';
 
+export type RemotePattern = {
+  /**
+   * Must be `http` or `https`.
+   */
+  protocol?: 'http' | 'https';
+
+  /**
+   * Can be literal or wildcard.
+   * Single `*` matches a single subdomain.
+   * Double `**` matches any number of subdomains.
+   */
+  hostname: string;
+
+  /**
+   * Can be literal port such as `8080` or empty string
+   * meaning no port.
+   */
+  port?: string;
+
+  /**
+   * Can be literal or wildcard.
+   * Single `*` matches a single path segment.
+   * Double `**` matches any number of path segments.
+   */
+  pathname?: string;
+};
+
 export interface Images {
   domains: string[];
+  remotePatterns?: RemotePattern[];
   sizes: number[];
   minimumCacheTTL?: number;
   formats?: ImageFormat[];
+  dangerouslyAllowSVG?: boolean;
+  contentSecurityPolicy?: string;
```

Comment:
These should be added to the type in `@vercel/build-utils` instead.

Reply:
Are the types listed twice in build-utils?

Reply:
I'm not sure what you mean "twice", but we have `Images` defined in build-utils that we should use instead: https://github.com/vercel/vercel/blob/main/packages/build-utils/src/types.ts#L355-L360

Reply:
Isn't that the same type/file?

Reply:
Yup. Nvm. I'm blind apparently 🤦